ocdaction/ocdaction/templates/challenge/challenge_list.html

```diff
@@ -18,26 +18,40 @@ <h2>My Challenge Hierarchy</h2>
                         </div>
-                        <div class="ibox-content">
-                            <table class="table table-striped">
-                                <thead>
-                                <tr>
-                                    <th class="ibox-content bold">Level</th>
-                                    <th class="ibox-content bold">Challenge</th>
-                                </tr>
-                                </thead>
-                                <tbody>
-                                    <tr>
-                                        <td class="text-align-left ibox-content">{% get_anxiety_level challenge %}</td>
-                                        <td class="text-align-left ibox-content">
-                                            <a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name }}</a>
-                                        </td>
-                                        <td class="text-align-right ibox-content"><a href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}"><span class="glyphicon glyphicon-hourglass margin-right-5"></span class="visuallyhidden">Continue</a></td>
-                                    </tr>
-                                </tbody>
-                            </table>
+                        <div class="ibox-content border-bottom">
+                            <div class="container">
+                                <div class="row">
+                                    <div class="col-xs-12 col-md-8 col-md-offset-2 ol-xl-6 col-xl-offset-3">
+                                        <table class="table table-striped">
+                                            <thead>
+                                            <tr>
+                                                <th class="ibox-content bold">Level</th>
+                                                <th class="ibox-content bold">Challenge</th>
+                                            </tr>
+                                            </thead>
+                                            <tbody>
+                                                <tr>
+                                                    <td class="text-align-left ibox-content col-xs-3">
+                                                        {% get_anxiety_level challenge %}
+                                                    </td>
+                                                    <td class="text-align-left ibox-content">
+                                                        <a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name }}</a>
+                                                    </td>
+                                                    <td class="text-align-right ibox-content"><a href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}"><span class="glyphicon glyphicon-hourglass margin-right-5"></span class="visuallyhidden">Continue</a></td>
```

Comment:
Missing a closing `span` tags. Let's break this line :)
```
<td class="text-align-right ibox-content">
    <a href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}">
        <span class="glyphicon glyphicon-play-circle margin-right-5"></span>
        <span class="visuallyhidden">Continue</span>
    </a>
</td>
```

Let's also change the 'hourglass' to a 'play-circle' to follow the design.